doc/BACKUP.md

```diff
@@ -394,14 +394,16 @@ it just gains the option to use a PostgreSQL database as well.
 If you just want to use PostgreSQL without using a cluster (for
 example, as an initial test without risking any significant funds),
 then after setting up a PostgreSQL database, you just need to add
-`--wallet=postgresql://${USER}:${PASSWORD}@${HOST}:${PORT}/${DB}`
+`--wallet=postgres://${USER}:${PASSWORD}@${HOST}:${PORT}/${DB}`
```

Comment:
Excellent catch, would love to cherry-pick this while we iterate on the broken link issue. Would you mind if I split this out into a separate PR?

Reply:
Cherry-pick for sure. I'll do the link in a sep. PR.